templates/test/ci/prow-topology/kustomization.yaml

```diff
@@ -5,9 +5,11 @@ resources:
   - ../../../flavors/clusterclass/cluster.yaml
   - cni-resource-set.yaml
   - ../../../addons/windows/csi-proxy/csi-proxy-resource-set.yaml
+  - ../../../addons/cluster-api-helm/calico.yaml
```

Comment:
I don't think you need to add it to both the clusterclass and the topology template since those work together

cc @willie-yao to correct me if I'm wrong

Reply:
I'm not too sure how ClusterClass would interact with add-ons seeing as ClusterClass does not explicitly support it in the code. It's possible that it will work anyway since HelmChartProxy is selecting Clusters and not owned by them. I'll try using it only in prow-topology for now to see if it works.

Reply:
Yes they do work together! Sorry for the late reply